docs/api/lockup/the-graph/02-structure.mdx

```diff
@@ -119,9 +119,10 @@ while also marking them as the sender. In that case, this initial address will b
 
 :::warning Warning: Deprecated
 
-Sablier V2.0 involved users deploying a "PRBProxy" contract through which they interacted with the Sablier contracts.
-With Sablier V2.1+, this is not the case any more as streams are now designed without any proxy contracts. Therefore,
-you don't need to worry about `proxenders` and `proxy` if you're not looking to support old versions of the protocol.
+Sablier Lockup V1.0 involved users deploying a "PRBProxy" contract through which they interacted with the Sablier
```

Comment:
Should we use npm versioning when referring to lockup versions? I made those changes in the other PR: https://github.com/sablier-labs/docs/pull/189

And here is a deployed version: https://v2-docs-isvsz5qf7-sablier.vercel.app/guides/lockup/versions/v1.1

Lmk if you agree as well. We can leave this PR as it is now and merge the versioning changes from the https://github.com/sablier-labs/docs/pull/189 into it.

Reply:
When referring to lockup contracts, as in the example you've highlighted, sure!

One caveat here: the subgraphs will still have to refer to "V20" or "V21" for backwards compatibility. There are still some places where I haven't updated all [references](https://docs.sablier.com/api/indexers/differences#versioning-and-lockup-flavors) to this versioning system since we have this discrepancy. Will keep thinking about a good approach.

Do we have a place in the docs where we explain this update we've made in versioning? Might be good to link to that in these places.

Reply:
Good idea about mentioning it in the docs. So far the answer is No. I can put it in the [Developer Guides](https://v2-docs-isvsz5qf7-sablier.vercel.app/guides/lockup/overview) in the Overview section. If you have any suggestion, lmk.

Reply:
That's good, I can then mention it from `/api` when talking about contract versions. Already adding caveats in the `@sablier/subraphs` repository.

Reply:
Added in https://github.com/sablier-labs/docs/pull/186/commits/c23c75cf164f04a33c3b5de31c88df5d5bc2cc1b. Can be seen [here](https://v2-docs-git-docs-flow-protocol-sablier.vercel.app/guides/lockup/overview).